realm/realm-library/src/main/java/io/realm/Realm.java

```diff
@@ -79,6 +79,19 @@
  * A standard pattern for working with Realm in Android activities can be seen below:
  * <p>
  * <pre>
+ * public class RealmApplication extends Application {
+ *
+ *     private static RealmConfiguration defaultRealmConfig;
```

Comment:
@cmelchior Do you think it would be even better we change the `defaultRealmConfig` to a local var?